slurm/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: enable required repos
+  include_tasks: base.yml
```

Comment:
@araji Where is `base.yml` located?

Reply:
will remove references to it and push again.
I added it to deploy what was missing for a slurm only target .will probably not be required after the refactoring.